### ISSUE TITLE
showCodeBlocks fails with NullPointerException

### ISSUE THREAD

**Issue by tloszabno** [renatoathaydes/spock-reports#103]
Hello,
Firstly, thanks  for that plugin ! 

Recently I found a problem with showCodeBlocks feature. 
Please look at code in https://github.com/tloszabno/spockpresentation (mvn3+j8) simply run mvn test.
When showCodeBlocks is enabled I getting NullPointerException: https://pastebin.com/PqNPkHwU
This exception occurs for each test. Even when I tried the simpler one with simple asserion (pl.tlos.spock.SpockTest).

Am I configured sth wrongly, or thats a plugin bug ? 
Thanks in advance for answer.

**Comment by renatoathaydes** [comment]
I am aware of this bug, but I am not sure in which circumstances it occurs.
Do you run your tests in parallel (this is one suspicion I have, parallel tests break this feature)?
Do you have any other idea about what triggers this bug?

**Comment by tloszabno** [comment]
I do not think parallel is problem, even running one single test with one feature it fails. Problem in my code is that It never worked for me (reports).

**Comment by renatoathaydes** [comment]
I checked out your project and confirmed the fix I have in the `next` branch works... but this happens because Spock can't get the source of your specs!! Trying to figure out why that is happening.

**Comment by renatoathaydes** [comment]
Ah!! I should've known that :) you have your tests in `src/test/java`, not the expected `src/test/groovy`!

You can configure this using this line in the config file:

```
com.athaydes.spockframework.report.testSourceRoots=src/test/java
```

I will make this a little bit more automatic.

**Comment by renatoathaydes** [comment]
Fixed in version 1.3.1. I made the documentation clearer about the need to set the directory where the test sources are to be found in case you don't use `src/test/groovy`.

**Comment by tloszabno** [comment]
Thanks ! I just test it and all works well :)
